frontend/src/theme/components/Tag.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export type TagVariants = 'solid' | 'light'
+
+export const Tag: ComponentMultiStyleConfig = {
+  parts: [],
+  baseStyle: {
+    label: {
+      textStyle: 'caption-1',
```

Comment:
textStyle cannot be used in this way; you might need to destructure:

```ts
import { textStyles } from './textStyles' // I forgot the real path/import
...
label: {
  ...textStyles['caption-1']
}
```

Reply:
wdyt about putting it on the component? not sure what's the appropriate way to do it but because it reads properly from the component level, i felt like it's better to have it there

Reply:
don't put it in the component. The example I gave was in the component theme. Just spread out the textStyle props in this file